### PR TITLE
MWPW-131779 : Quote block automation test scripts

### PR DESCRIPTION
**Pull request by skumar09**
[MWPW-131779](https://jira.corp.adobe.com/browse/MWPW-131779): Quote block automation test scripts

PR includes:
- Automation test scripts for the following features of the Quote block
  - Quote
  - Quote (contained)
  - Quote (inline, contained)
  - Quote (borders, contained)
  - Quote (contained, align-right)
  - Quote (xl-spaced)

### CODE REVIEW

**Review by JackySun9 — APPROVED** [review]

File: features/milo/quote.block.spec.js
Comment (amauch-adobe) [inline features/milo/quote.block.spec.js:1]: Did you want to have tags for each type of quote block style as you did for Marquee?

File: tests/milo/quote.block.test.js
Comment (amauch-adobe) [inline tests/milo/quote.block.test.js:1]: I think it would be good to check images are visible as well since that is apart of the quote block.

**Review by amauch-adobe — CHANGES_REQUESTED** [review]
Approved, just some ideas, feel free to respond back to them.

File: tests/milo/quote.block.test.js
Comment (amauch-adobe) [reply tests/milo/quote.block.test.js:1]: blast, I must have missed it.  Thanks.

File: features/milo/quote.block.spec.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
         figCaption: 'Rapha Abreu',
         cite: 'Global Vice President of Design, Coca-Cola Company',
       },
-      tags: '@Quote @smoke @regression @milo,',
+      tags: '@Quote @quote-inine @smoke @regression @milo,',
```
Comment (amauch-adobe) [inline features/milo/quote.block.spec.js:37]: I think the tag is supposed to be "@quote-inline", it seems to be missing the 'l'.

**Review by amauch-adobe — APPROVED** [review]
Approved, see my small grammar comment. ;)

**Pull request merged**